src/target/mm-preview.js

```diff
@@ -69,40 +69,34 @@ const preview = {
 
     const allFiles = await globPromise(`${data.inputDir.replace(/\\/g, '/')}/**/*`);
 
-    const filesArray = allFiles
-      .filter((file) => fs.lstatSync(file).isFile())
-      .map((file) => {
-        return {
-          filePath: file,
-          s3Path: data.outputDir + path.relative(data.inputDir, file).replace(/\\/g, '/'),
-          contentType: mime.lookup(file),
-          content: fs.readFileSync(file),
-        };
-      });
+    const filesArray = await Promise.all(
```

Comment:
Now you have a promise.all in promise.all and map 2 times, is this necessary? can't we do one promise.all and one map instead?

Reply:
great catch, updated the code